app/views/check_csv_file.scala.html

```diff
@@ -15,40 +15,66 @@
  *@
 
 @import models.upscan.UpscanInitiateResponse
-@import views.html.includes.upscan_file_upload_form
 @import config.ApplicationConfig
 @import _root_.utils.ERSUtil
-@(scheme: String, invalidChars: String, fileId: String)(implicit request: Request[AnyRef], flash: Flash, messages: Messages, upscanResponse: UpscanInitiateResponse, appConfig: ApplicationConfig, ersUtil: ERSUtil)
-@* check return file Template *@
 
-@import _root_.views.html.templates._
+@this(ers_main: views.html.ers_main,
+        upscan_file_upload_form: views.html.includes.upscan_file_upload_form,
+        govukButton: GovukButton,
+        govukErrorSummary: GovukErrorSummary,
+        govukBackLink: GovukBackLink,
+        govukFileUpload: GovukFileUpload,
+        hmrcPageHeading: HmrcPageHeading
+)
 
+@(scheme: String, invalidChars: String, fileId: String)(implicit request: Request[AnyRef], flash: Flash, messages: Messages, upscanResponse: UpscanInitiateResponse, appConfig: ApplicationConfig, ersUtil: ERSUtil)
+@* check return file Template *@
 @scripts = {
-	<script>var INVALID_CHARACTERS=/@Html(invalidChars)/</script>
-	<script src='@routes.Assets.at("javascripts/ers_csv_scripts.js")'> </script>
+    <script>var INVALID_CHARACTERS =/@Html(invalidChars)/</script>
+    <script src='@routes.Assets.at("javascripts/ers_csv_scripts.js")'> </script>
+    <script>document.getElementById("check-file-button").disabled = true</script>
```

Comment:
Duplication: this is present on each file upload page, in ers_csv_scripts and ers_scripts.

Reply:
removing it now